spyder/utils/help/tutorial.rst

```diff
@@ -179,31 +165,31 @@ Inspecting objects defined in the console
         Print "Hello World" and return None
```

Comment:
We need a full stop after ``None`` (as this has been added to the docstring in line 38 above), i.e. change ``return None`` to ``return None.``

Reply:
Thanks @fangohr for chiming in.

@CAM-Gerlach?

Reply:
Hey Professor @fangohr , thanks! Good catch; once you're done reviewing it I can submit a PR and fix everything you find. Thanks!

Reply:
Also, just FYI #7659 has the up to date version.

Reply:
Fixed three different instances as requested. Thanks!